enzyme/Enzyme/TypeAnalysis/TypeTree.h

```diff
@@ -371,10 +372,18 @@ class TypeTree : public std::enable_shared_from_this<TypeTree> {
 
     if (Result.minIndices.size() > EnzymeMaxTypeDepth) {
       Result.minIndices.pop_back();
-      if (EnzymeTypeWarning)
-        llvm::errs() << "not handling more than " << EnzymeMaxTypeDepth
-                     << " pointer lookups deep dt:" << str() << " only(" << Off
-                     << "): " << str() << "\n";
+      if (EnzymeTypeWarning) {
+        if (orig) {
```

Comment:
@vchuravy having a custom c abi handler here for backtracing on the julia level would probably be the way to go.